src/components/widgets/eve-mail/eve-mail.js

```diff
@@ -0,0 +1,60 @@
+import React, {Component} from 'react';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {eveMailFetchHeaders} from '../../../actions/eve-mail';
+import axios from 'axios';
+const EVE_PIC = require('../../../assets/eve-login.png');
+
+class EveMail extends Component {
+  constructor(props, context){
+    super(props, context);
```

Comment:
do not pass props to super unless you're using `this.props` within the `constructor`. do not pass `context` under any circumstances